app/dashboard/static/js/app/view-tests-board-job-kernel.2017.7.2.js

```diff
@@ -518,12 +519,14 @@ require([
         gFileServer = document.getElementById('file-server').value;
     }
 
-    gLabList = document.getElementById('lab-names');
-    if (gLabList !== null) {
-        gLabList = String(gLabList.value).split(',');
+    function extendLabList(lab) {
+        gLabList.push(lab.value)
+    }
 
+    Array.prototype.slice.call(document.querySelectorAll('input.lab-names'))
+        .map(extendLabList);
+    if (gLabList) {
         if (gLabList.length > 0) {
```

Comment:
Minor tweak.

You can remove the previous check on `gLabList` and only check on its length. In Javascript an empty array will still evaluate to true, but since you are already setting it up as an array at the beginning of the file, you can just check for the length.